docker/Dockerfile_dali_tf

```diff
@@ -3,6 +3,15 @@ FROM ${TF_CUSTOM_OP_BUILDER_IMAGE}
 
 WORKDIR /opt/dali_tf_build
 
+ARG NVIDIA_DALI_BUILD_FLAVOR
```

Comment:
I think you need to adjust https://github.com/NVIDIA/DALI/blob/master/docker/build.sh#L169 as well.

Reply:
you are right. Done